testData/grouping/groupingTemplate.json

```diff
@@ -0,0 +1,45 @@
+[
+    {
+        "name": "Windows Hight Contrast",
```

Comment:
Typo "High"

Reply:
Typo remains

Reply:
Ah, thank you! We corrected this typo along with the linting warnings in the ``PSP3-coverage`` branch which also includes Tony's test code coverage work.